spec/jobs/prepare_person_notifications_job_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+RSpec.describe PreparePersonNotificationsJob, type: :job do
+  subject(:perform) { described_class.new.perform(topic_id: person.id, action_name: action_name) }
+
+  let(:person) { create :person }
+  let!(:move1) { create :move, person: person }
+  let!(:move2) { create :move, person: person }
+  let!(:other_move) { create :move }
+
+  before do
+    allow(PrepareMoveNotificationsJob).to receive(:perform_now)
+    perform
+  end
+
+  context 'when updating a person' do
+    let(:action_name) { 'update' }
+
+    describe 'it calls PrepareMoveNotificationsJob for the related moves' do
+      it { expect(PrepareMoveNotificationsJob).to have_received(:perform_now).with(topic_id: move1.id, action_name: action_name) }
+      it { expect(PrepareMoveNotificationsJob).to have_received(:perform_now).with(topic_id: move2.id, action_name: action_name) }
+      it { expect(PrepareMoveNotificationsJob).not_to have_received(:perform_now).with(topic_id: other_move.id, action_name: action_name) }
+    end
+  end
+
+  context 'when creating a person' do
+    let(:action_name) { 'create' }
```

Comment:
What do you think of removing this context block? The action_name doesn't actually affect how the service behaves? It just gets passed through to the other service?

Reply:
hmm, interesting question: so higher up (in the controller) there is a decision that creating a person should not email, but updating a person should email. I guess I wanted to capture in this test that the `PreparePersonNotificationsJob` behaves the same for both creates and updates.I think its important to test both contexts, but I'll refactor it a bit. 👍